tests/test_ui.py

```diff
@@ -179,6 +196,24 @@ def addSample(self):
         widget_2 = self.getDockedWidget(self.window.docks, InsertPrimitiveDialog.dock_flag)
         self.assertIsNot(widget, widget_2)
         old_vertex_count = len(self.model.sample.vertices)
+
+        self.window.sample_properties_dialog_action.trigger()
+        sample_properties_widget = self.getDockedWidget(self.window.docks, SampleProperties.dock_flag)
+        self.assertTrue(isinstance(sample_properties_widget, SampleProperties))
+        bytes_used = self.model.sample.vertices.nbytes
+        bytes_to_mb_factor = 1 / (1024**2)
```

Comment:
you already defined this above

Reply:
second definition of bytes_to_mb_factor removed from test_ui.py